src/wp_rsa_asym.c

```diff
@@ -318,6 +318,14 @@ static int wp_rsaa_encrypt(wp_RsaAsymCtx* ctx, unsigned char* out,
                 ok = 0;
             }
         }
+        else if (ctx->padMode == RSA_NO_PADDING) {
+            rc = wc_RsaDirect((byte *)in, (word32)inLen, out,
+                (word32 *)&outSize, wp_rsa_get_key(ctx->rsa),
```

Comment:
Don't use outSize.
Use a word32 type variable and assign result into outSize.
(Big endian bad but also top bits aren't cleared for little endian.)